client.go

```diff
@@ -150,7 +151,14 @@ func NewSnowthClient(discover bool, addrs ...string) (*SnowthClient, error) {
 		sc.AddNodes(node)
 		sc.ActivateNodes(node)
 		sc.Logger.Debugf("activated node: %s -> %s", addr, state.Identity)
+		numActiveNodes++
 	}
+
+	if numActiveNodes == 0 {
+		err := errors.New("No snowth nodes could be activated")
```

Comment:
No problems with this, but the assignment is redundant, since we are returning here.
```suggestion
```